services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -353,6 +353,72 @@ public static Builder builder() {
   @Nullable
   public abstract String exclude();
 
+  /**
+   * A list of exclude.
```

Comment:
Don't think this is needed. Redundant with the `@return` and there is already a description
```suggestion
```